cloudbuild.yaml

```diff
@@ -2,18 +2,26 @@ timeout: 1200s
 steps:
   - name: gcr.io/cloud-builders/docker
     args:
+      - HOME=/root
```

Comment:
https://github.com/kubernetes/test-infra/tree/master/config/jobs/image-pushing#simple-build-example
"    # default cloudbuild has HOME=/builder/home and docker buildx is in /root/.docker/cli-plugins/docker-buildx
    # set the home to /root explicitly to if using docker buildx
    # - HOME=/root"